File: my-shop-selenium/src/main/java/data/dto/AddressDTO.java
package data.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class AddressDTO {

    private String firstName;
    private String lastName;
    private String company;
    private String address1;
    private String address2;
    private String city;
    private String state;
    private String zip;
    private String country;
    private String homePhone;
    private String mobilePhone;
    private String additionalInfo;
    private String addressAlias;

    public AddressDTO(String firstName, String lastName, String address1, String city, String state, String zip, String country, String homePhone, String addressAlias) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.address1 = address1;
        this.city = city;
        this.state = state;
        this.zip = zip;
        this.country = country;
        this.homePhone = homePhone;
        this.addressAlias = addressAlias;
    }
}

File: my-shop-cypress/cypress/e2e/data/dto/address-dto.ts
export default class AddressDTO {

    firstName: string;
    lastName: string;
    company: string;
    address1: string;
    address2: string;
    city: string;
    state: string;
    zip: string;
    country: string;
    homePhone: string;
    mobilePhone: string;
    additionalInfo: string;
    addressAlias: string;

    constructor(firstName: string, lastName: string, address1: string, city: string, state: string, zip: string, country: string, homePhone: string, addressAlias: string) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.address1 = address1;
        this.city = city;
        this.state = state;
        this.zip = zip;
        this.country = country;
        this.homePhone = homePhone;
        this.addressAlias = addressAlias;
    }
}